.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: node_js
 node_js:
   - "6"
-  - "7"
```

Comment:
Does this mean we no longer support node 7?

Reply:
The travis build is just used to make sure the code builds and the tests pass. We don't have internal users on node 7 as far as I know, so I made the change to make the builds a bit faster. I'll still publish the artifacts for node 7 to the internal repo for now, but I'd like to just support node 6 for the old stable build and node 10 which is what nodequark will be using soon. I don't know if we'll be able to get there soon though.